omf-python/tests/test_category_attribute.py

```diff
@@ -0,0 +1,90 @@
+import json
+import omf_python
+from os import path
+from unittest import TestCase
+
+class TestCategoryAttribute(TestCase):
+    def setUp(self) -> None:
+        omf_dir = path.join(path.dirname(__file__), "data")
+        one_of_everything = path.join(omf_dir, "one_of_everything.omf")
+        self.reader = omf_python.Reader(one_of_everything)
+
+    def test_should_return_categories_attribute(self) -> None:
+        # Given I get the attributes for my PointSet
+        attributes = self.reader.project.elements[1].attributes
+
+        # Then I should have three attributes
```

Comment:
I'd prefer we don't have comments that just describe what the code below is obviously doing, it just makes it harder to read and I end up skipping over the comments IMO.

Reply:
Maybe consider splitting into more than one test if its testing a lot of different things